packages/core/src/meta/MetaManager.js

```diff
@@ -42,6 +42,8 @@ export default class MetaManager {
    * @param {string} name Meta information property name, for example
    *        {@code keywords}.
    * @param {string} value The meta information value.
+   *
+   * @param {Object=} other_attrs Additional attributes to be used in the meta tag (some types of meta tags require them).
```

Comment:
Can we change the `other_attrs` to camelCase format `otherAttrs` so it follows the same code style as the rest of the code base?